### PR TITLE
NodeJS support

### PR DESCRIPTION
**Pull request by anton-st**

### CODE REVIEW

**Comment by anton-st** [comment]
Code was modified to match changes in squash...

File: src/extension.ts [hunk 1]
```diff
@@ -379,7 +379,8 @@ class SquashExtention {
     }
 
     async waitForDebugConfigWithImage(image: string, dbgr: string): Promise<string> {
-        const result = await squash<squashinterface.DebugRequest>(`debug-request ${image} ${dbgr}`);
+        let pname = get_conf_or("process-name", "")
+        const result = await squash<squashinterface.DebugRequest>(`debug-request ${image} ${dbgr} ${pname}`);
```
Comment (yuval-k) [inline src/extension.ts]: this should be -processname, right?

**Review by yuval-k — CHANGES_REQUESTED** [review]

File: src/extension.ts (hunk 1 above)
Comment (anton-st) [reply src/extension.ts]: Right now full config option name will be vs-squash.process-name and it seems to comply with others.
In general, we have 2 naming conventions there:
vs-squash.squash-path
vs-squash.remotePath

**Comment by anton-st** [comment]
Fixed

**Comment by anton-st** [comment]
Also, plz review python support...

**Review by yuval-k — APPROVED** [review]

**Pull request merged**